### ISSUE TITLE
Add required-attribute

### ISSUE THREAD

**Issue by NicTorgersen** [LouisMazel/vue-phone-number-input#49]
The input element doesn't have the required-attribute, only aria-required, so right now this component does not support HTML5 form validation.

**Comment by LouisMazel** [comment]
It's fixed into `0.2.7` version

[0.2.7](https://github.com/LouisMazel/vue-phone-number-input/releases/tag/0.2.7)
